src/models/searchservice/response/directanswer/FeaturedSnippetDirectAnswer.ts

```diff
@@ -12,14 +10,6 @@ import { Snippet } from './Snippet';
 export interface FeaturedSnippetDirectAnswer<T = unknown> extends DirectAnswer<T> {
   /** {@link DirectAnswerType}.FeaturedSnippet. */
   type: DirectAnswerType.FeaturedSnippet,
-  /** {@inheritDoc DirectAnswer.value} */
-  value?: T,
-  /** {@inheritDoc DirectAnswer.relatedResult} */
-  relatedResult: Result,
-  /** {@inheritDoc DirectAnswer.verticalKey} */
-  verticalKey: string,
-  /** {@inheritDoc DirectAnswer.fieldType} */
-  fieldType: BuiltInFieldType | string,
```

Comment:
since davish told us the fieldType here can only be rich_text or multi_line_text should we reflect that here? I'm not sure whether we should also include unknown here but leaning towards not needing it?

Reply:
yea, updating